presto-main/src/main/java/io/prestosql/sql/planner/optimizations/PredicatePushDown.java

```diff
@@ -486,6 +484,22 @@ public PlanNode visitJoin(JoinNode node, RewriteContext<Expression> context)
                 }
             }
 
+            DynamicFiltersResult dynamicFiltersResult = createDynamicFilters(node, equiJoinClauses, session, idAllocator);
+            Map<String, Symbol> dynamicFilters = dynamicFiltersResult.getDynamicFilters();
+            leftPredicate = combineConjuncts(leftPredicate, combineConjuncts(dynamicFiltersResult.getPredicates()));
+
+            PlanNode leftSource;
+            PlanNode rightSource;
+            boolean equiJoinClausesEquivalent = ImmutableSet.copyOf(equiJoinClauses).equals(ImmutableSet.copyOf(node.getCriteria()));
```

Comment:
I would rename it to `equiJoinClausesUnmodified`, `equivalent` suggest they could have the same semantics but might use different symbols.